components/AdForm/types.js

```diff
@@ -13,15 +14,20 @@ export const positions = {
   ],
   single: [
     // 'before item 1',
+    'before navbar',
     'before content',
     ...range(1, 17).map(
       n => `after ${MIN_LIMIT_VALUE * n} characters in content`,
     ),
     'after content',
     // ...range(1, 21).map(n => `after item ${n}`)
+    'after comments',
+    'after next posts',
+    'after more in 1',
```

Comment:
As commented here https://github.com/frontity/saturn-theme/pull/204#discussion_r245582641 `after more in 1` doesn't give enough information of where it is.